src/TensorFlowNET.Core/Keras/Activations/Activations.cs

```diff
@@ -26,9 +27,65 @@ public class Activation
         }
     }
 
+    /// <summary>
+    /// The ActivationAdaptor is used to store string, Activation, and Func for Laysers Api to accept different types of activation parameters.
+    /// One of the properties must be specified while initializing.
+    /// </summary>
+    public class ActivationAdaptor
+    {
+        /// <summary>
+        /// The name of activaiton function, such as `tanh`, `sigmoid`.
+        /// </summary>
+        public string? Name { get; set; } = null;
+
+        /// <summary>
+        /// The available Activation instance of activaiton function, such as keras.activations.Tanh, keras.activations.Sigmoid.
+        /// </summary>
+        public Activation? Activation { get; set; } = null;
```

Comment:
`IActivation` is the interface of any `Activation`.

Reply:
Thanks for figuring out these mistakes. I'll resubmit soon. : )